sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsOperationStatus.cs

```diff
@@ -11,5 +11,8 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("State")]
     public partial struct TextAnalyticsOperationStatus
     {
+        /// <summary> Added for compilation. </summary>
+        /// TODO: delete later
+        public static TextAnalyticsOperationStatus Rejected { get; } = new TextAnalyticsOperationStatus(CancellingValue);
```

Comment:
could u add an issue for this so we don't forget to clean it up? it needs to happen during this release cycle

Reply:
Issue: https://github.com/Azure/azure-sdk-for-net/issues/28326

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/28326